### PR TITLE
created other entities

### PR DESCRIPTION
**Pull request by S00189802**

### CODE REVIEW

File: backend-service/src/main/java/com/powerrangers/backendservice/entity/Bookings.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+
+import lombok.*;
+
+
+@Data
+@Entity
+public class Bookings {
```
Comment (Clancinio) [inline backend-service/src/main/java/com/powerrangers/backendservice/entity/Bookings.java:16]: Bookings -> Booking

**Review by Clancinio — CHANGES_REQUESTED** [review]

File: backend-service/src/main/java/com/powerrangers/backendservice/entity/Bookings.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  Long id;
+
+  @NonNull
+  //@ManyToOne(targetEntity= Room.class) **************WHAT DO WE ADD HERE?*******************
```
Comment (S00189591) [inline backend-service/src/main/java/com/powerrangers/backendservice/entity/Bookings.java:23]: Not good practice to leave in comment. If you're not using it get rid of it

File: backend-service/src/main/java/com/powerrangers/backendservice/entity/Slot.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  //@ManyToOne(targetEntity= Room.class) **************WHAT DO WE ADD HERE?*******************
+  Long roomID;
+
+  @NonNull
+  LocalTime timeStart;
+
+  @NonNull
+  LocalTime timeEnd;
+
+
```
Comment (vinithanagu) [inline backend-service/src/main/java/com/powerrangers/backendservice/entity/Slot.java:32]: can we remove unwanted white space ?

File: backend-service/src/main/java/com/powerrangers/backendservice/entity/RoomProperty.java [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  @NonNull
+  String equipment;
+
+  @NonNull
+  Integer capacity;
+
+  @NonNull
+  String imageUrl;
+
+
```
Comment (vinithanagu) [inline backend-service/src/main/java/com/powerrangers/backendservice/entity/RoomProperty.java:38]: same here white space

**Review by vinithanagu — CHANGES_REQUESTED** [review]

File: backend-service/src/main/java/com/powerrangers/backendservice/entity/Bookings.java (hunk 2 above)
Comment (TheRobinGuy) [reply backend-service/src/main/java/com/powerrangers/backendservice/entity/Bookings.java]: If it's something you want to revisit, you can leave a //TODO: something

File: backend-service/src/main/java/com/powerrangers/backendservice/entity/Bookings.java (hunk 1 above)
Comment (Clancinio) [reply backend-service/src/main/java/com/powerrangers/backendservice/entity/Bookings.java]: Actually, it should be postfixed with Entity, and same for the others. So, BookingEntity....

**Review by TheRobinGuy — APPROVED** [review]

**Review by S00189591 — APPROVED** [review]

**Pull request merged**